packages/checkout-extension/src/handlers/createGetConsignmentHandler.ts

```diff
@@ -0,0 +1,22 @@
+import { ExtensionCommandType, ExtensionMessageType } from '@bigcommerce/checkout-sdk';
+
+import { CommandHandler, HandlerProps } from './CommandHandler';
+
+export function createGetConsignmentHandler({
```

Comment:
Do we need this handler? Since I envision this to be called from extension lib to SDK?

Reply:
If it's a command based communication, we need a command handler on the `checkout-js` side.